frontend/src/components/ArtifactPreview.tsx

```diff
@@ -74,9 +79,7 @@ const ArtifactPreview: React.FC<ArtifactPreviewProps> = ({
   );
 
   if (!storage) {
-    return (
-      <Banner message={'Can not retrieve storage path from artifact uri: ' + value} mode='error' />
-    );
+    return value ? <div>{value}</div> : <div>No artifact URI is found.</div>;
```

Comment:
nit: Artifact **URI** should always be available, it may not be a storage URL though. From that perspective, the original message seems okay except the `mode='error'` part? Up to you do decide what's the best non-error message here.